pyscitt/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 PACKAGE_NAME = "pyscitt"
-PACKAGE_VERSION = "0.5.0"
+PACKAGE_VERSION = "1.0.0"
```

Comment:
Any reason for making this the first stable release?

Reply:
I think holding on until we get a little more user feedback would be best. We could move to 0.6.0?

Reply:
CLI contains breaking changes, please suggest something else

Reply:
Until a first stable release, it is possible to increment the minor version number even for breaking changes. This is standard for semantic version numbers used for software versioning. See https://semver.org/#spec-item-4

Reply:
(FWIW, we already had breaking changes in the CLI in the past and we always incremented the minor number accordingly)

Reply:
done